ui_framework/doc_site/src/views/header/header.html

```diff
@@ -43,12 +43,58 @@
             <span class="kuiHeader__notification">3</span>
           </button>
         </div>
-        <div class="kuiHeaderItem">
+        <div class="kuiHeaderItem kuiPopMenu kuiPopMenu--anchorRight" data-id="toggleButton">
           <button class="kuiHeaderItem__button">
             <svg class="kuiIcon kuiIcon--medium">
               <use href="#apps" />
             </svg>
           </button>
+          <div class="kuiPopMenu__body kui--textAlignCenter kui--fontSizeSmall kui--bottomShadow" data-id="toggleMenu">
```

Comment:
It seems like `kui--textAlignCenter` `kui--fontSizeSmall` should be scoped to the individual DialMenuItems themselves, since the PopMenu is just a container and we could decide to other things inside of it at some point. This will also make it easier to reuse.